src/book/02-projects/01-nubots/03-build-system.mdx

```diff
@@ -21,6 +21,7 @@ To build the NUbots code a couple of programs are needed on the host machine
 - Git: You need this clone and work on the NUbots repository
 - Docker: On Linux and Mac you can just install Docker (and set up user permissions) and you are good to go. On Windows you will also need WSL
 - Python3: A `requirements.txt` file exists in the root of the NUbots repository and in the `nuclear` folder. Install the required python modules using `sudo -H pip3 install requirements.txt` and `sudo -H pip3 install nuclear/requirements.txt`
+- Visual Studio Code: recommended, but not required
```

Comment:
Do we also want to add information about the containerised development with vscode?

Reply:
I haven't played with it much, can you add some details?